CHANGELOG.md

```diff
@@ -4,58 +4,62 @@
 
 - Chg #161: Allow to reuse manager test code in related packages (@arogachev)
 - New #161: Add `ManagerInterface` (@arogachev)
-- Chg #161: Raise PHP version to 8.0 (@arogachev)
+- Chg #161, #217: Raise PHP version to 8.1 (@arogachev)
 - Bug #178: Exclude parent role from `Manager::getAllChildRoles()` (@arogachev)
 - Enh #134: Improve handling and control of `Assignment::$createdAt` (@arogachev)
 - Chg #134: Add `$createdAt` parameter to `ManagerInterface::assign()` (@arogachev)
-- Chg #134: Replace parameters with `$assignment` parameter in `AssignmentsStorageInterface::add()` (@arogachev)
-- Enh #165: Improve performance (@arogachev)
+- Chg #134: Replace all parameters with single `$assignment` parameter in `AssignmentsStorageInterface::add()` 
+  (@arogachev)
+- Enh #165, #206: Improve performance:
+  - Optimize calls for getting child items within the loops.
+  (@arogachev)
```

Comment:
```suggestion
- Enh #165, #206: Improve performance: optimize calls for getting child items within the loops (@arogachev)
```

Reply:
I removed the list, but rewrote it a bit, because it's not the only one change.